src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
```

Comment:
you don't need writing it here, just put all import statements to `main.scss`

Reply:
If I use @import or @use in main.css with these files they are't being picked up?